styles/main.css

```diff
@@ -204,31 +211,37 @@ header a:hover {
 }
 main {
       display:grid;
-      grid-emplate-columns: auto;
+      grid-template-columns: 1fr 1fr;

```

Comment:
You need to specify the size of each column.  The declaration you gave just leaves it up to the browser to choose the size and number of columns